docs/source/user/settings.rst

```diff
@@ -204,6 +204,16 @@ API and user token related settings
 
 These settings control details related to the API and the radius user token.
 
+``OPENWISP_RADIUS_CSV_BASEURL``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
what does not convince me is the fact that we're adding the base url only for the CSV files, why is that?
And why for the fw-upgrader module the setting is more generic?

Reply:
Even for Firmware upgrader the name I've given is `OPENWISP_FIRMWARE_IMAGE_BASEURL`,
I'll need to make it generic, but starting here to give a generic name! :smile: